src/components/page/sectionBreak.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImageLoader from '../site/imageLoader';
+import Block from './blocks/index';
+
+class SectionBreak extends React.PureComponent {
+  render() {
+    const { title, blocksGroups, shared: blockShared } = this.props;
+    const sectionBreakImagePath = '/images/section_break_celebration.gif';
+
+    return (
+      <div className="section-break">
+        <ImageLoader
+          src={sectionBreakImagePath}
+          alt="Animated image of fireworks"
+        />
```

Comment:
The fireworks gif img needs to be set to `width: 100%`.  As is, it is overflowing as soon as its container is narrower than its max width.

Reply:
added `image-responsive` class that sets:

```
max-width: 100%;
height: auto;
```

and a prop to set it in `ImageLoader`